src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -79,7 +80,10 @@ def handle_result(self, subscription: UptimeSubscription | None, result: CheckRe
         if subscription is None:
             # If no subscription in the Postgres, this subscription has been orphaned. Remove
             # from the checker
-            send_uptime_config_deletion(result["subscription_id"])
+            # TODO: Send to region specifically from this check result once we update the schema
+            send_uptime_config_deletion(
+                get_active_region_configs()[0].slug, result["subscription_id"]
```

Comment:
Right now this is just using the only region that exists. Will this break if we add more in a weird or order or anything

Reply:
It'll have weird behaviour if we add more than one, but once we start sending the region with the results we can remove this hack and it'll be ok

Reply:
cool cool